.gitignore

```diff
@@ -1,9 +1,39 @@
 vendor
 .vendor-new
-.vscode
 debug
 .env
 dist
 coverage*.out
 mobile-security-service
 !cmd/mobile-security-service
+.vscode/*
+!.vscode/launch.json
+
+
+## ui/.gitignore
+
+# dependencies
+ui/node_modules
+ui/.pnp
+ui/.pnp.js
+
+# testing
+ui/coverage
+
+# production
+ui/build
+
+# misc
+ui/.DS_Store
+ui/.env.local
+ui/.env.development.local
+ui/.env.test.local
+ui/.env.production.local
+
+ui/npm-debug.log*
+ui/yarn-debug.log*
+ui/yarn-error.log*
+ui/yarn.lock
+
+ui/.vscode/*
+!ui/.vscode/launch.json
```

Comment:
Can we use some grepping patterns above, to make the .gitignore less verbose, ie `*.log`, etc

Reply:
@damienomurchu not sure what change you are suggesting here. The goal of line 38 and 39 is to only include launch.json from the .vscode folder.

Reply:
@laurafitzgerald actually meant the increasingly verbose `.gitignore` file, not just lines 38/ 39 (github only seems to let me comment on one line :( ). What I was trying to say was that we might be able to reduce the number of entries added to the `.gitignore` by using grepping patterns, ie like adding `*.log` to ignore all .log files rather than adding each.log file to be ignored, if that makes sense? :)